packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -404,7 +404,7 @@ export type HooksTree = Array<HooksNode>;
 
 let mostLikelyAncestorIndex = 0;
 
-function findSharedIndex(hookStack, rootStack, rootIndex) {
+function findSharedIndex(hookStack, rootStack, rootIndex: number) {
```

Comment:
There are a bunch that are not annotate like these. What's the criteria? Are these `: any` or something?

Reply:
This was the flow provided codemod and for the first pass I removed a lot that were inferred as either `any` or complex types (not understanding that the type was `Fiber`)